src/main/java/uk/gov/justice/digital/delius/controller/advice/SecureControllerAdvice.java

```diff
@@ -97,7 +98,7 @@
 
     @ExceptionHandler(InvalidRequestException.class)
     public ResponseEntity<ErrorResponse> handleException(final InvalidRequestException e) {
-        log.debug("Bad Request (400) returned", e);
+        log.debug("Bad Request (400) returned with message: {}", e.getMessage());
```

Comment:
Do we definitely want to suppress all 4XX exceptions?  I can see why for 401, 403, 404 as they are BAU, but 400/409 indicate the client has a problem which we might need to highlight?